README.md

```diff
@@ -134,6 +134,8 @@ Execute `mvn test` with the (optional) flags outlined below:
 * `-DchangeObjects=createTable,dropTable` flag allows you to run specific changeObjects rather than all. Use comma
  separated lists.
 * `-DchangeData=insert,delete` flag that allows to run specific changeData through ChangeDataTests. Use comma separated list
+* `-DconfigFile=customConfigFile.yml` enables to override default config file which is(`src/test/resources/harness-config.yml`)
```

Comment:
@jnewton03 will such flags be helpful for you? To override harness-config.yml and run platforms by prefix